src/tabs/TabsV2.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {Button} from "../forms/button";
+import {ClassNames} from "./ClassNames";
+import * as utils from "../utils";
+import {Dropdown, DropdownMenu, DropdownItem, MenuItemType} from "../forms/dropdown";
+import {TabPane} from "./TabPane";
+
+/**
+ * props for tabs
+ * @param {tabs} List of all TabsV2
+ * @param {id} string to identify tab group uniquely
+ * @param {tabOrientation} orientation of tab either vertical or horizontal
+ * @param {tabClickCallback} callback function from parent to call on tab click
+ * @param {overflowTabsfrom} name of tab from which tabs added to overflow menu.
+ *         optional prop, if set activates overflow tabs
+ * @param {dataqa} quality engineering tag
+ */
+type TabsV2Prop = {
+    tabs: TabV2Details[];
+    id: string;
+    tabOrientation: TabV2Orientation;
+    tabClickCallback?: (evt: React.MouseEvent<HTMLElement>, clickedTab: TabV2Details) => void;
```

Comment:
Try to keep props same as Tabs component, so that we can adapt this without too many changes. 
tabClickCallback -> should be onTabClick only.

Reply:
Done